### PR TITLE
Try TravisCI Windows

### PR DESCRIPTION
**Pull request by erikdubbelboer**

### CODE REVIEW

File: fs_test.go [hunk 1]
```diff
@@ -740,6 +741,10 @@ func TestServeFileContentType(t *testing.T) {
 }
 
 func TestServeFileDirectoryRedirect(t *testing.T) {
+	if runtime.GOOS == "windows" {
+		t.SkipNow()
+	}
```
Comment (erikdubbelboer) [inline fs_test.go:746]: This test fails because of `/` vs `\`.

File: prefork/README.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ Preforks master process between several child processes increases performance, b
 
 **WARNING: using prefork prevents the use of any global state!. Things like in-memory caches won't work.**
 
+Note: Prefork is not supported on Windows.
```
Comment (erikdubbelboer) [inline prefork/README.md:9]: See: https://golang.org/pkg/os/exec/#Cmd.ExtraFiles

File: .travis.yml [hunk 3]
```diff
@@ -9,14 +9,13 @@ dist: bionic
...
   - osx
+  - windows
 go:
   - tip
   - 1.14.x
   - 1.13.x
   - 1.12.x
   - 1.11.x
-  - 1.10.x
-  - 1.9.x
```
Comment (erikdubbelboer) [inline .travis.yml:19]: These versions doesn't support everything on windows. They are also multiple years old so I think we can drop support.

File: prefork/README.md (hunk 2 above)
Comment (savsgio) [reply prefork/README.md:9]: I think so with reuseport (reuseaddr) is supported

**Pull request merged**